lib/dag.js

```diff
@@ -0,0 +1,140 @@
+var stream = require('stream')
+var multihash = require('multihashes')
+var base58 = require('base58-native')
+var merkledag = require('node-ipfs-mdag')
```

Comment:
Not suppose to be there, I was using it, and might go back to it, but for now it wasn't doing what I wanted. I guess I forgot to delete it.